qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -504,26 +561,40 @@ def new_cpmg_sequence(
 def new_uhrig_sequence(
     duration=None, number_of_offsets=None, pre_post_rotation=False, **kwargs
 ):
-    """
-    Creates Uhrig Single Axis Sequence.
+    r"""
+    Creates the Uhrig sequence.
 
     Parameters
     ---------
     duration : float
-        Total duration of the sequence. Defaults to None
+        Total duration of the sequence :math:`\tau`. Defaults to None.
     number_of_offsets : int, optional
-        Number of offsets. Defaults to None
+        Number of offsets. Defaults to None.
     pre_post_rotation : bool, optional
-        If True, a :math:`\\pi.2` rotation is added at the
+        If True, a :math:`X_{\pi/2}` rotation is added at the
         start and end of the sequence.
     kwargs : dict
-        Additional keywords required by
-        qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
+        Additional keywords required by DynamicDecouplingSequence.
 
     Returns
     -------
-    qctrlopencontrols.dynamical_decoupling_sequences.DynamicDecouplingSequence
-        Uhrig (single-axis) sequence
+    DynamicDecouplingSequence
+        The Uhrig sequence.
+
+    Notes
+    -----
+    The Uhrig sequence [#]_ is parameterized by duration :math:`\tau` and number of
+    offsets :math:`n`. The sequence consists of :math:`Y_{\pi}` operations at offsets given by
+
+    .. math::
+        t_i = \tau \sin^2 \left( \frac{i\pi}{2(n+1)} \right) \;,
+
+    where :math:`i = 1, \cdots, n`.
+
+    References
+    ----------
+    .. [#] `Götz S. Uhrig, Physical Review Letters 98, 100504 (2007)
```

Comment:
```suggestion
    .. [#] `G. S. Uhrig, Physical Review Letters 98, 100504 (2007)
```

I think Physical Review usually uses just the initial of the names of the authors in the references